inc/saitam.h

```diff
@@ -469,7 +469,12 @@ typedef enum _sai_tam_int_type_t
     /**
      * @brief Direct Export (aka postcard)
      */
-    SAI_TAM_INT_TYPE_DIRECT_EXPORT
+    SAI_TAM_INT_TYPE_DIRECT_EXPORT,
+
+    /**
+     * @brief Telemetry data at the end of the packet
+     */
+    SAI_TAM_INT_TYPE_TAIL_TIMESTAMP,
```

Comment:
make it more specific, IFA?

Reply:
Sure, Also the same comment from Mickey in the community discussion, will change to be more specific.